pkg/kubelet/kubelet_pods.go

```diff
@@ -356,6 +361,7 @@ func nodeHostsFileContent(hostsFilePath string, hostAliases []v1.HostAlias) ([]b
 	if err != nil {
 		return nil, err
 	}
+	hostsFileContent = append([]byte(managedHostsHeaderWithHostNetwork), hostsFileContent...)
```

Comment:
it's a little confusing to prepend managedHostsHeaderWithHostNetwork this way... would read better if that was set first in the function and every change was an append (switching to the buffer.Write...() approach used in managedHostsFileContent would probably make sense)